### PR TITLE
Chat improvements #71

### PR DESCRIPTION
**Pull request by Michal-Szczepaniak**

### CODE REVIEW

File: qml/components/MessageItem.qml [hunk 1]
```diff
@@ -109,7 +109,9 @@ ListItem {
             rightMargin: Theme.paddingMedium
         }
 
-        text: created_at
+        text: (parseInt(Qt.formatDateTime(created_at, "d")) < parseInt(Qt.formatDateTime(new Date(), "d"))) ?
```
Comment (almindor) [inline qml/components/MessageItem.qml]: Could you put this into a function?

**Review by almindor — CHANGES_REQUESTED** [review]
Just coz I'll need it for others too :) you can put it in js/common for example.

File: qml/components/MessageItem.qml (hunk 1 above)
Comment (Michal-Szczepaniak) [reply qml/components/MessageItem.qml]: Sure thing

**Comment by Michal-Szczepaniak** [comment]
Done :)

**Review by almindor — APPROVED** [review]

**Comment by almindor** [comment]
Oh sorry, I forgot, you'll need to sign your commit.

See [here](https://help.github.com/en/articles/signing-commits) for how to set it up. I think best is to get the signing up and then remake the PR. Sorry for being a PITA but it is after all a security related project.

**Pull request closed without merging**